php/php.editor/src/org/netbeans/modules/php/editor/verification/Bundle.properties

```diff
@@ -31,6 +31,7 @@ ImmutableVariablesCustomizer.jLabel1.text=Number of Allowed Assignments per Vari
 csl-hints/text/x-php5/hints/braces=Braces
 csl-hints/text/x-php5/hints/psr0=PSR-0: Autoloading Standard
 csl-hints/text/x-php5/hints/psr1=PSR-1: Basic Coding Standard
+csl-hints/text/x-php5/hints/psr4=PSR-4: Autoloader
```

Comment:
Should not it be "Autoloading Standard", similarly to PSR-0?

Reply:
"Autoloader" is official name of PSR-4 - https://www.php-fig.org/psr/psr-4/

Reply:
OK.